app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
You are modifying the `body` element across different files, which is not a good idea. This is a global element that can easily mess up the styles on your site. My suggestion is to group all changes related to global elements like `html` and `body` on a single file.